client/landing/gutenboarding/onboarding-block/create-site/index.tsx

```diff
@@ -51,7 +52,12 @@ const CreateSite: FunctionComponent< {} > = () => {
 					/>
 					<div className="create-site__progress-numbered-steps">
 						{ progressSteps.map( ( _, index ) => (
-							<p>{ `Step ${ index + 1 } of ${ progressSteps.length }` }</p>
+							<p>
+								{ sprintf( __( 'Step %(currentStep)d of %(totalSteps)d' ), {
```

Comment:
Oops, I commented on commit instead of PR: https://github.com/Automattic/wp-calypso/commit/ffad542580cc9b3228b9679f51c9a85012bec118#r38366105

I wasn't sure if we should use `_n()` here.